Download.Mvc/Startup.cs

```diff
@@ -40,6 +41,8 @@ public void ConfigureServices(IServiceCollection services)
 
             // Add framework services.
             services.AddMvc();
+
+            services.AddSingleton<ITitleCodesService, TitleCodesService>();

```

Comment:
@laholstege I'm thinking this should be AddTransient . But you can double check with @srkirkland